students/17th/team1/gyuboshim/westagram/user/views.py

```diff
@@ -0,0 +1,68 @@
+from django.shortcuts import render
+
+import json
+from django.views   import View
+from django.http    import HttpResponse, JsonResponse
+
+from .models        import User
+
+PASSWORD_LEN_LIMIT = 8
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data            =   json.loads(request.body)
+            user_data       =   data['user']
+            phone_number    =   user_data['phone_number']
+            email_adress    =   user_data['email_adress']
+            name            =   user_data['name']
+            nickname        =   user_data['nickname']
+            password        =   user_data['password']
+
+
+            if user_data['email_adress'] and user_data['password']:
+                
+                if user_data['email_adress'] == User.objects.filter(email_adress = email_adress):
```

Comment:
```suggestion
                if User.objects.filter(email_adress = email_adress):
```
둘에 차이가 있는지 한번 생각해 봅시다.

Reply:
아하 이해했습니다!

Reply:
한가지 질문이 있습니다. 
if User.objects.filter(email_adress = email_adress).extists() 와
if User.objects.filter(email_adress = email_adress)의 차이는 없는건가요..? 

찍어봤는데, 가독성을 위해서 쓰는것같다라는 생각밖에 들지않아서 혹시 다른 차이가 있나해서 질문합니다.

Reply:
if 자체는 동작합니다. 하지만 리턴은 다르죠.